apps/docs/content/guides/storage/iceberg-buckets.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: 'Iceberg Buckets'
+subtitle: ''
+---
+
+<Admonition type="note">
+    This feature is currently in Private Beta and subject to change as the API evolves.
+</Admonition>
+
+Iceberg Buckets are specialized Supabase Storage buckets powered by Apache Iceberg, an open-table format for efficiently managing analytical datasets.
```

Comment:
The opening "why Iceberg" bit might be a bit overly technical, it kind of assumes you already know why you would want data in Iceberg format. Not sure what the audience for this feature is -- it might well be only those people who already want the feature, but if we also want people to consider it who have never heard of Iceberg, we might need a softer opener:

- Your regular Supabase tables in Postgres are for your app
- For analytics, you need other data stores for (a) efficient analytics (b) cheap storage (c) not interfering with your production database operations

Or something like that, I'm sure you know the value proposition better 😄